packages/react-aria-components/src/Button.tsx

```diff
@@ -65,7 +90,11 @@ export interface ButtonProps extends Omit<AriaButtonProps, 'children' | 'href' |
   /** Submitted as a pair with the button's value as part of the form data. */
   name?: string,
   /** The value associated with the button's name when it's submitted with the form data. */
-  value?: string
+  value?: string,
+  /**
+   * Whether to disable events immediately and display a `renderPendingState` after a `pendingDelay`.
```

Comment:
This description kinda makes it sound like `renderPendingState` and `pendingDelay` are props that Button accepts but those don't exist